static/js/entry/dashboard.js

```diff
@@ -1,11 +1,14 @@
+require('react-hot-loader/patch');
```

Comment:
This ensures that the code that bootstraps the react-hot-loader is executed before anything else. it spews a warning otherwise.